### PR TITLE
dev/accessibility#3: Add accessible label on Contact Edit form elements and helpicon

### PR DESCRIPTION
**Pull request by monishdeb**
Overview
----------------------------------------
The PR adds aria-label attribute to all those form elements which don't have accessible labels and also adds this attribute on helpicon links

Before
----------------------------------------
Form elements like Website, Email, IM, Phone etc. don't have accessible labels and thus when tested with WAVE tool, let's say Website field:
screen shot 2018-06-28 at 11 55 17 pm

After
----------------------------------------
Adds aria-label on form elements like Website, Email, IM, Phone etc.
screen shot 2018-06-28 at 11 53 26 pm

### CODE REVIEW

**Comment by seamuslee001** [comment]
Jenkins re test this please

File: js/model/crm.uf.js [hunk 1]
```diff
@@ -32,7 +32,8 @@
    * Add a help link to a form label
    */
   function addHelp(title, options) {
-    return title + ' <a href="#" onclick=\'CRM.help("' + title + '", ' + JSON.stringify(options) + '); return false;\' title="' + ts('%1 Help', {1: title}) + '" class="helpicon"></a>';
+    var title = ts('%1 Help', {1: title});
```
Comment (seamuslee001) [inline js/model/crm.uf.js]: @monishdeb just get rid of `var` here and that should fix the style issue.

**Comment by colemanw** [comment]
This looks good to me.

**Comment by colemanw** [comment]
@monishdeb I'm curious to know why `aria-label` was needed on the help icon link. Was it not sufficient to have a `title`?

**Pull request merged**